.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Feature Request
+about: Suggest a new feature for the rippled project
+title: "[Title with short description] (Version: [rippled version])"
+labels: Feature Request
+assignees: ''
+
+---
+<!-- Please search existing issues to avoid creating duplicates.-->
+
+## Description
+### Describe the feature you'd like
```

Comment:
Remove this subheading. Remember, instructions should be in comments, and there is already a comment here. Non-comments should be fixed strings we don't expect users to edit.